common/scala/src/main/scala/whisk/core/connector/Message.scala

```diff
@@ -100,7 +101,15 @@ case class CompletionMessage(
 
 object CompletionMessage extends DefaultJsonProtocol {
     def parse(msg: String) = Try(serdes.read(msg.parseJson))
-    implicit val serdes = jsonFormat2(CompletionMessage.apply)
+    implicit val serdes = jsonFormat3(CompletionMessage.apply)
+}
+
+case class InvocationFinishedMessage(name: String, successful: Boolean) extends Message {
+    override def serialize = InvocationFinishedMessage.serdes.write(this).compactPrint
+}
+
+object InvocationFinishedMessage extends DefaultJsonProtocol {
+    implicit val serdes = jsonFormat(InvocationFinishedMessage.apply _, "name", "successful")
```

Comment:
Why is this defined in `Messages`? After all it seems like only being used by the `InvokerPool` and thus should go there?